Lombiq.Tests.UI/MonkeyTesting/GremlinsScripts.cs

```diff
@@ -33,9 +42,22 @@ internal string Build()
                 string speciesPart = Species != null ? string.Join(", ", Species) : null;
 
                 string mogwaisPart = Mogwais != null ? string.Join(", ", Mogwais) : null;
-
                 return
+
 @$"(function() {{
+    const customLogger = {{
+        log: function () {{
+            console.log.apply(null, arguments);
+            if (['click', 'dblclick', 'mouseup'].some(x => arguments[2] === x)) {{
```

Comment:
Please use a proper variable name instead of `x`.

Reply:
Renamed.